### PR TITLE
Add entry for Nexon America

### PR DESCRIPTION
**Pull request by ghost**
Add Nexon America to Gaming

### CODE REVIEW

File: _data/gaming.yml [hunk 1]
```diff
@@ -203,6 +203,13 @@ websites:
     twitter: MojangSupport
     img: mojang.png
 
+  - name: Nexon America
+    url: https://www.nexon.com/main/en
+    img: nexon.png
+    tfa:
```
Comment (Rfosu2k2) [inline _data/gaming.yml:209]: You need to put the word "Yes" for the value of "tfa:" since you are indicating that they do support TFA.

Comment (ghost) [reply _data/gaming.yml:209]: Sorry, I don't quite understand. Do you mean I should put a `tfa: yes`? I don't see this on other entries in the file, and in the local preview, I see it listed with the check mark in the "software" field.

Screenshot 2019-10-15 at 21 08 26

Comment (Rfosu2k2) [reply _data/gaming.yml:209]: You are correct, please disregard.

File: _data/gaming.yml [hunk 2]
```diff
@@ -203,6 +203,13 @@ websites:
...
+      - software
```
Comment (Carlgo11) [inline _data/gaming.yml]:
```suggestion
      - proprietary
```

File: _data/gaming.yml [hunk 3]
```diff
@@ -203,6 +203,13 @@ websites:
     twitter: MojangSupport
     img: mojang.png
 
+  - name: Nexon America
```
Comment (Carlgo11) [inline _data/gaming.yml]: From what I see there's nothing different between the login systems for the US version or any other version.
```suggestion
  - name: Nexon
```

File: _data/gaming.yml [hunk 4]
```diff
@@ -203,6 +203,13 @@ websites:
...
+    url: https://www.nexon.com/main/en
```
Comment (Carlgo11) [inline _data/gaming.yml]:
```suggestion
    url: https://www.nexon.com
```

File: _data/gaming.yml [hunk 5]
```diff
@@ -203,6 +203,13 @@ websites:
...
+    img: nexon.png
+    tfa:
+      - software
+    doc: https://support.nexon.net/hc/en-us/articles/360025920992-Authenticator-FAQ
```
Comment (Carlgo11) [inline _data/gaming.yml]:
```suggestion
    doc: https://support.nexon.net/hc/en-us/articles/360025920992
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I suggest using this image instead:
nexon logo

File: _data/gaming.yml (hunk 3 above)
Comment (ghost) [reply _data/gaming.yml]: I can't confirm for other regions, unfortunately. If using "Nexon" as a whole instead is fine, sure.

File: _data/gaming.yml (hunk 2 above)
Comment (ghost) [reply _data/gaming.yml]: It appears to be a generic TOTP (Google Authenticator based), however. I registered it and get generic 6-number codes.

**Review by ghost — COMMENTED** [review]
Image change is OK.

File: _data/gaming.yml (hunk 2 above)
Comment (Carlgo11) [reply _data/gaming.yml]: The documentation page talks about "The Authenticator" so I thought it was some kind of app that they had build.
Which 2FA app(s) have you tried with Nexon?

Comment (ghost) [reply _data/gaming.yml]: I set it up using Authy, which provides the aforementioned 6-number codes. Google Authenticator is explicitly recommended; you can add accounts using the provided QR code or a manual code. I have attached images to reference these statements.

Screenshot 2019-10-16 at 18 58 35
Screenshot 2019-10-16 at 18 59 15

**Comment by ghost** [comment]
Sorry, meant to combine the changes.

File: _data/gaming.yml (hunk 2 above)
Comment (Carlgo11) [reply _data/gaming.yml]: Alright. Thanks!
Here's a better fitting change then:
```suggestion
      - totp
```

**Comment by ghost** [comment]
> I suggest using this image instead:
> nexon logo

Would you like me to commit this myself, or would you prefer changing/committing it yourself?

**Comment by Carlgo11** [comment]
If you'd do it that'd be great.

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**